### PR TITLE
Mention that Mullvad accepts prepaid cards

### PR DESCRIPTION
**Pull request by akosnikhazy**
Changes proposed in this PR:

- mention of prepaid cards at Mullvad

<summary>

- [ ] I agree to the terms listed below:
  <details><summary>Contribution terms (click to expand)</summary>
  1) I am the sole author of this work.
  2) I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
  3) I have disclosed any relevant conflicts of interest in my post.
  4) I agree to the Community Code of Conduct.
  </details>

### CODE REVIEW

File: docs/vpn.md [hunk 1]
```diff
@@ -242,7 +242,7 @@ Mullvad provides the source code for their desktop and mobile clients in their [
 
 #### :material-check:{ .pg-green } Accepts Cash and Monero
 
-Mullvad, in addition to accepting credit/debit cards and PayPal, accepts Bitcoin, Bitcoin Cash, **Monero** and **cash/local currency** as anonymous forms of payment. They also accept Swish and bank wire transfers.
+Mullvad, in addition to accepting credit/debit cards and PayPal, accepts Bitcoin, Bitcoin Cash, **Monero** and **cash/local currency** as anonymous forms of payment, prepaid card with redeem code are also available. They also accept Swish and bank wire transfers.
```
Comment (rollsicecream) [inline docs/vpn.md:245]:
```suggestion
Mullvad, in addition to accepting credit/debit cards and PayPal, accepts Bitcoin, Bitcoin Cash, **Monero** and **cash/local currency** as anonymous forms of payment, prepaid card with redeem codes are also available. They also accept Swish and bank wire transfers.
```
Aside of that, looks good to me.

**Review by rollsicecream — CHANGES_REQUESTED** [review]

File: docs/vpn.md (hunk 1 above)
Comment (WayTooWill) [reply docs/vpn.md:245]: Could just be me but there may also need to be a change from "prepaid card" to prepaid cards".

**Review by WayTooWill — COMMENTED** [review]
Could just be me but there may also need to be a change from "prepaid card" to prepaid cards".

**Comment by rollsicecream** [comment]
> Could just be me but there may also need to be a change from "prepaid card" to prepaid cards".

Yeah, you're right, I just updated my changes to reflect that :+1:

**Review by dngray — APPROVED** [review]

**Pull request merged**